cosmo_tester/resources/blueprints/openstack-start-vm/blueprint.yaml

```diff
@@ -0,0 +1,133 @@
+  flavor:
+    type: string
+  image_id:
+    type: string
+  key_pair_path:
+    type: string
+
+node_templates:
+  external_network:
+    type: cloudify.openstack.nodes.Network
+    properties:
+      use_external_resource: true
+      resource_id: public
+      openstack_config: &OPENSTACK_CONFIG
+        username: { get_input: os_username }
+        password: { get_input: os_password }
+        tenant_name: { get_input: os_tenant_name }
+        auth_url: { get_input: os_auth_url }
+        region: { get_input: os_region }
+
+  router:
+    type: cloudify.openstack.nodes.Router
+    properties:
+      resource_id: { concat: [{ get_input: prefix }, '-simple-vm-router'] }
+      openstack_config: *OPENSTACK_CONFIG
+    relationships:
+      - target: external_network
+        type: cloudify.relationships.connected_to
+
+  network:
+    type: cloudify.openstack.nodes.Network
+    properties:
+      resource_id: { concat: [{ get_input: prefix }, '-simple-vm-network'] }
+      openstack_config: *OPENSTACK_CONFIG
+
+  subnet:
+    type: cloudify.openstack.nodes.Subnet
+    properties:
+      resource_id: { concat: [{ get_input: prefix }, '-simple-vm-subnet'] }
+      subnet:
+        ip_version: 4
+        cidr: 10.67.79.0/24
+      openstack_config: *OPENSTACK_CONFIG
+    relationships:
+      - target: network
+        type: cloudify.relationships.contained_in
+      - target: router
+        type: cloudify.openstack.subnet_connected_to_router
+
+  floating_ip:
+    type: cloudify.openstack.nodes.FloatingIP
+    properties:
+      floatingip:
+        floating_network_name: { get_property: [external_network, resource_id] }
+      openstack_config: *OPENSTACK_CONFIG
+
+  security_group:
+    type: cloudify.openstack.nodes.SecurityGroup
+    properties:
+      resource_id: { concat: [{ get_input: prefix }, '-simple-vm-security-group'] }
+      rules:
+      rules:
+        - port: 80
+          remote_ip_prefix: 0.0.0.0/0
+        - port: 22
+          remote_ip_prefix: 0.0.0.0/0
+        - port: 8080
+          remote_ip_prefix: 0.0.0.0/0
+        - port: 8101
+          remote_ip_prefix: { get_property: [subnet, subnet, cidr] }
+        - port: 5555
+          remote_ip_prefix: { get_property: [subnet, subnet, cidr] }
+        - port: 5672
+          remote_ip_prefix: { get_property: [subnet, subnet, cidr] }
+        - port: 53229
+          remote_ip_prefix: { get_property: [subnet, subnet, cidr] }
+      openstack_config: *OPENSTACK_CONFIG
+
+  keypair:
+    type: cloudify.openstack.nodes.KeyPair
+    properties:
+      resource_id: { concat: [{ get_input: prefix }, '-simple-vm-keypair'] }
+      private_key_path: { get_input: key_pair_path }
+      openstack_config: *OPENSTACK_CONFIG
+
+  simple_server:
+    type: cloudify.openstack.nodes.Server
+    properties:
+      #resource_id: { get_input: manager_server_name }
+      install_agent: false
+      server:
+        image: { get_input: image_id }
+        flavor: { get_input: flavor }
```

Comment:
move the image and flavor properties outside of 'server'. to be directly under the 'properties' section (a new convention we started, the openstack plugin supports it)